backend/LexData/Entities/UserEntityConfiguration.cs

```diff
@@ -13,6 +13,7 @@ public override void Configure(EntityTypeBuilder<User> builder)
         base.Configure(builder);
         builder.Property(u => u.LocalizationCode).HasDefaultValue(User.DefaultLocalizationCode);
         builder.Property(u => u.Username).UseCollation(LexBoxDbContext.CaseInsensitiveCollation);
+        builder.HasIndex(u => u.Username).IsUnique();
```

Comment:
we should also add the FK mapping for UsersICreated and created by.

Reply:
Done in commit e6051813.